lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -659,13 +683,22 @@ class ClassMetadataInfo implements ClassMetadata
      */
     public $changeTrackingPolicy = self::CHANGETRACKING_DEFERRED_IMPLICIT;
 
+    /**
+     * A Flag indicating whether one or more columns of this class have to be reloaded
+     * after insert / update operations.
+     *
+     * @readonly
```

Comment:
This means the property can only be written in the constructor, and cannot be used here for that reason.

Reply:
Ah, i was about to suppress them, so its readonly to all the other places in the code, I saw that you reverted it, fine by me.